File: 2019/templates/template.cpp
#include <algorithm>
#include <iostream>
#include <string>
#include <vector>
#include <queue>
#include <set>
#include <map>

using namespace std;

string solution(int N, vector<int> ciper){
    return "Hello";
}

int main() {
	int test_cases;
	cin >> test_cases;

	for (int test_case = 1; test_case <= test_cases; test_case++) {
		int N, L;
		cin >> N >> L;
		vector<int> ciper(L, 0);
		for(int i=0;i<L;i++){
		    cin >> ciper[i];
		}
		string answer = solution(N, ciper);
		cout << "Case #" << test_case << ": " << answer << endl;
		cout.flush();
	}
	return 0;
}

File: 2019/templates/template.go
package main

import (
	"bytes"
	"fmt"
)

func solution(n string) string {
	var out bytes.Buffer
	for _, char := range n {
		if char == 'E'{
			out.WriteRune('S')
		} else {
			out.WriteRune('E')
		}
	}
	return out.String()
}

func main() {
	var testCases int
	fmt.Scanln(&testCases)

	for testCase := 1; testCase <= testCases; testCase++ {
		var n string
		fmt.Scan(&n)

		answer := solution(n)
		fmt.Printf("Case #%d: %s\n", testCase, answer)
	}
}
